packages/assets/lib/extensions/index.tsx

```diff
@@ -22,29 +22,41 @@ export const Extensions: Record<string, ExtensionConfig> = {
   enkrypt: {
     title: "Enkrypt",
     website: "enkrypt.com",
+    networksSupported: "*",
+    features: "*",
   },
   "fearless-wallet": {
     title: "Fearless Wallet",
     website: "fearlesswallet.io",
+    networksSupported: "*",
+    features: "*",
   },
   "metamask-polkadot-snap": {
     title: "MetaMask Polkadot Snap",
     website: [
       "snaps.metamask.io",
       "snaps.metamask.io/snap/npm/chainsafe/polkadot-snap",
     ],
+    networksSupported: ["polkadot", "kusama", "westend"],
```

Comment:
We should not have this param. We should allow the wallets react to a non supported network with error message. Or else we will end up maintaining any changes that a wallet decides to do

Reply:
Okay, removed ✅